src/main/java/org/broadinstitute/hellbender/engine/filters/FragmentLengthReadFilter.java

```diff
@@ -3,14 +3,19 @@
 import org.broadinstitute.hellbender.cmdline.Argument;
 import org.broadinstitute.hellbender.utils.read.GATKRead;
 
+import java.io.Serializable;
+
 /**
  * Keep reads that are within a given max fragment length.
  */
-public final class FragmentLengthReadFilter implements ReadFilter {
+public final class FragmentLengthReadFilter extends ReadFilter implements Serializable  {
     private static final long serialVersionUID = 1l;
 
-    @Argument(fullName = "maxFragmentLength", shortName = "maxFragment", doc="Keep only read pairs with fragment length at most equal to the given value", optional=true)
-    public int maxFragmentLength = 1000000;
+    @Argument(fullName = "maxFragmentLength",
+            shortName = "maxFragment",
+            doc = "Keep only read pairs with fragment length at most equal to the given value",
+            optional = false)
+    public Integer maxFragmentLength; // no default value, since the command line parser will think its set
```

Comment:
Can we allow args in plugins to have default values? This arg, for example, should definitely be optional, as it was originally, as we don't want to require the user to always specify `maxFragmentLength` when using this filter. Should be possible if you add a callback to your descriptor at the point in the arg parsing where it injects values into filter instances, to keep track of which filters had args specified (needed for post-parsing validation when you build the final list of selected filters)